src/commands/sessions.ts

```
@@ -0,0 +1,226 @@
+		const sessionData = await loadSessionWindowData({
+			since: ctx.values.since,
+			until: ctx.values.until,
+			claudePath: ctx.values.path,
+			mode: ctx.values.mode,
+			order: ctx.values.order,
+			sessionLimit: ctx.values.sessionLimit,
+			warningThreshold: ctx.values.warningThreshold,
+		});
+
+		if (sessionData.length === 0) {
+			if (ctx.values.json) {
```

ESLint

[error] Unexpected any value in conditional. An explicit comparison or type conversion is required.

(ts/strict-boolean-expressions)